Documentation/common-issues.md

```diff
@@ -383,3 +385,34 @@ Uncordon the node:
 ```
 kubectl uncordon <node-name>
 ```
+
+# Rook Agent modprobe exec format error
+## Symptoms
+* PersistentVolumes from Ceph fail/timeout to mount
+* Rook Agent logs contain `modinfo: ERROR: could not get modinfo from 'rbd': Exec format error` lines
+
+## Solution
+If it is feasible to upgrade your kernel, you should upgrade to `4.x`, even better is >= `4.7` due to a feature for CephFS added to the kernel.
+
+If you are unable to upgrade the kernel, you need to go to each host that will consume storage and run:
+```
+modprobe rbd
+```
+This command inserts the `rbd` module into the kernel.
+
+To persist this fix, you need to add the `rbd` kernel module to either `/etc/modprobe.d/` or `/etc/modules-load.d/`.
+For both paths create a file called `rbd.conf` with the following content:
+```
+rbd
+```
+Now when a host is restarted, the module should be loaded automatically.
+
+# Using multiple shared filesystem (CephFS) is attempted on a kernel version older than 4.7
+## Symptons
+* Mounting a filesystem other than the first created, will cause the first filesystem to be always mounted instead
```

Comment:
i would break this into more than 1 line:
```
* More than one shared filesystem (CephFS) has been created in the cluster
* A pod attempts to mount any other shared filesystem besides the **first** one that was created
* The pod incorrectly gets the first filesystem mounted instead of the intended filesystem
```